### PR TITLE
koordlet: add some logs for cpi and psi metric cache

### PR DESCRIPTION
**Pull request by zwzhang0107**
### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/koordlet/metriccache/storage_test.go [hunk 1]
```diff
@@ -848,6 +856,14 @@ func Test_storage_ContainerPSIMetric_CRUD(t *testing.T) {
 				}
 			}
 
+			gotNum, err := s.CountContainerPSIMetric()
```
Comment (saintube) [inline pkg/koordlet/metriccache/storage_test.go:859]: To count the metrics individually, we may create the metric cache via `newStorage()` while the `NewStorage` would open the same DB.

Comment (zwzhang0107) [reply pkg/koordlet/metriccache/storage_test.go:861]: both are the same db, and needs defer close now.

**Review by jasonliu747 — APPROVED** [review]
/lgtm

**Pull request merged**